### PR TITLE
Revert "Merge pull request #954 from Azure/pr-920-provision-telemetry"

### PR DESCRIPTION
**Pull request by boumenot**
This reverts commit 154a6ec208608223a83d749a58a0bbffa5798001, reversing
changes made to b943cbc7067fbb8500a6a2ca1c7588f803ff7c81.

### CODE REVIEW

**Comment by boumenot** [comment]
I made this change with the intent of tracking down a difference between the VM's report CPU/Memory and the CPU/Memory of the VM size the customer bought. Unfortunately, this change was unnecessary. The CPU/Memory as seen by the guest is already reported, and this event was superfluous.  I missed this fact, and so did my reviewer.

The code is unnecessary, which is why I am

File: azurelinuxagent/pa/provision/default.py [hunk 1]
```diff
@@ -269,53 +264,12 @@ def deploy_ssh_keypairs(self, ovfenv):
             logger.info("Deploy ssh key pairs.")
             self.osutil.deploy_ssh_keypair(ovfenv.username, keypair)
 
-    def report_event(self, message, is_success=False, duration=0,
-                     operation=WALAEventOperation.Provision):
+    def report_event(self, message, is_success=False, duration=0):
```
Comment (jasonzio) [inline azurelinuxagent/pa/provision/default.py]: I would recommend leaving this one change (the ability to specify the operation name) in place. You might not need it now, but you've already written and tested it, and there's zero cost to carrying it along should you need it in the future.

**Comment by boumenot** [comment]
I liked your idea.  I preserved the parameter, and I kept the two tests.
